### PR TITLE
SEOImageUpdates: User Story 1463880, Small PR 3

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Make a smaller PR.

Fixes #1463880 

Contributes to #10839

### CODE REVIEW

File: docs/framework/winforms/advanced/how-to-obtain-font-metrics.md [hunk 1]
```diff
@@ -24,20 +24,20 @@ The <xref:System.Drawing.FontFamily> class provides the following methods that r
   
  The following illustration shows the various metrics.
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-obtain-font-metrics.md]:
```suggestion
 The following illustration shows the various metrics:
```

File: docs/framework/winforms/advanced/how-to-obtain-font-metrics.md [hunk 2]
```diff
@@ -24,20 +24,20 @@ The <xref:System.Drawing.FontFamily> class provides the following methods that r
   
  The following illustration shows the various metrics.  
   
- ![Fonts Text](./media/fontstext7a.png "fontstext7A")  
+ ![Illustration of various font metrics](./media/how-to-obtain-font-metrics/various-font-metrics.png "Illustration of various font metrics")  
   
 ## Example  
  The following example displays the metrics for the regular style of the Arial font family. The code also creates a <xref:System.Drawing.Font> object (based on the Arial family) with size 16 pixels and displays the metrics (in pixels) for that particular <xref:System.Drawing.Font> object.  
   
  The following illustration shows the output of the example code.
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-obtain-font-metrics.md]:
```suggestion
 The following illustration shows the output of the example code:
```

File: docs/framework/winforms/advanced/how-to-obtain-font-metrics.md [hunk 3]
```diff
@@ -24,20 +24,20 @@ The <xref:System.Drawing.FontFamily> class provides the following methods that r
...
- ![Fonts Text](./media/fontstext7a.png "fontstext7A")  
+ ![Illustration of various font metrics](./media/how-to-obtain-font-metrics/various-font-metrics.png "Illustration of various font metrics")  
   
 ## Example  
  The following example displays the metrics for the regular style of the Arial font family. The code also creates a <xref:System.Drawing.Font> object (based on the Arial family) with size 16 pixels and displays the metrics (in pixels) for that particular <xref:System.Drawing.Font> object.  
   
  The following illustration shows the output of the example code.  
   
- ![Fonts Text](./media/csfontstext8.png "csFontsText8")  
+ ![Example code output of Arial font](./media/how-to-obtain-font-metrics/example-output-code-arial-font.png "Example code output of Arial font")
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-obtain-font-metrics.md]:
```suggestion
 ![Example code output of Arial font metrics.](./media/how-to-obtain-font-metrics/example-output-code-arial-font.png)  
```

File: docs/framework/winforms/advanced/how-to-obtain-font-metrics.md [hunk 4]
```diff
@@ -24,20 +24,20 @@ The <xref:System.Drawing.FontFamily> class provides the following methods that r
   
  The following illustration shows the various metrics.  
   
- ![Fonts Text](./media/fontstext7a.png "fontstext7A")  
+ ![Illustration of various font metrics](./media/how-to-obtain-font-metrics/various-font-metrics.png "Illustration of various font metrics")
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-obtain-font-metrics.md]:
```suggestion
 ![Illustration of font metrics: ascent, descent, and line spacing.](./media/how-to-obtain-font-metrics/various-font-metrics.png)  
```

File: docs/framework/winforms/advanced/how-to-obtain-font-metrics.md [hunk 5]
```diff
@@ -24,20 +24,20 @@ The <xref:System.Drawing.FontFamily> class provides the following methods that r
...
   
- ![Fonts Text](./media/csfontstext8.png "csFontsText8")  
+ ![Example code output of Arial font](./media/how-to-obtain-font-metrics/example-output-code-arial-font.png "Example code output of Arial font")  
   
  Note the first two lines of output in the preceding illustration. The <xref:System.Drawing.Font> object returns a size of 16, and the <xref:System.Drawing.FontFamily> object returns an em height of 2,048. These two numbers (16 and 2,048) are the key to converting between font design units and the units (in this case pixels) of the <xref:System.Drawing.Font> object.  
   
  For example, you can convert the ascent from design units to pixels as follows:  
   
- ![Fonts Text](./media/fontstext9.png "FontsText9")  
+ ![Example of converting from design units to pixels](./media/how-to-obtain-font-metrics/convert-font-units-example.png "Example of converting from design units to pixels")
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-obtain-font-metrics.md]:
```suggestion
 ![Formula showing the conversion from design units to pixels](./media/how-to-obtain-font-metrics/convert-font-units-example.png)  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @damabe. A few things to address before merging.

**Comment by damabe** [comment]
@maira Feedback incorporated. Updated CATS report:

https://contentqacats.azurewebsites.net/CheckResults/2019051600045/Report/

**Review by mairaw — APPROVED** [review]

**Pull request merged**